app/views/tag/_topicCard.html.erb

```diff
@@ -5,7 +5,7 @@
     <div class="card shadow-sm p-2 mb-3 bg-white rounded" >
       <div class="card-header mt-2" style=" padding:0.8em; background-color: inherit; border:none;">
         <% if current_user && current_user.following(tag.name) && controller.controller_name != 'tag' %>
-          <a style="margin-top:-8px;" data-method="delete" data-confirm="Are you sure you'd like to unfollow this topic?" rel="tooltip" title="<%= t('tag.show.unfollow') %>" class="pull-right" href="/unsubscribe/tag/<%= tag.name %>"><i style="color:#ccc;" class="fa fa-times"></i></a>
+          <a style="margin-top:-8px;" data-method="delete" data-confirm="Are you sure you'd like to unfollow this topic?" rel="tooltip" title="<%= t('tag.show.unfollow') %>" class="pull-right" href="/unsubscribe/tag/<%= tag.name %>" data-remote="true" onclick="window.location.reload(true);"><i style="color:#ccc;" class="fa fa-times"></i></a>
```

Comment:
ah, clever! Does this work both if it succeeds or fails?

Reply:
I tried this, but it doesn't work. The page gets reloaded without toggling unfollow to follow.

Reply:
Ah, sorry, so this PR is not quite ready then? Would you like to try a different approach to this? Perhaps we have to add a `return_to` parameter to the subscription controller, and make it possible to redirect back?https://github.com/publiclab/plots2/blob/master/app/controllers/subscription_controller.rb#L27-L81